istioctl/cmd/testdata/auth/upgrade/multiple-access-rules-one-subject.golden.yaml

```diff
@@ -0,0 +1,54 @@
+apiVersion: security.istio.io/v1beta1
+kind: AuthorizationPolicy
+metadata:
+  name: default-all
+  namespace: default
+spec:
+  rules:
+  - from:
+    - source:
+        namespaces:
+        - istio-system
+    to:
+    - operation:
+        methods:
+        - GET
+---
+apiVersion: security.istio.io/v1beta1
+kind: AuthorizationPolicy
+metadata:
+  name: default-productpage
```

Comment:
Okay for this PR but I think we probably need a better naming to avoid conflict later.